egs/multi_zh-hans/ASR/RESULTS.md

```diff
@@ -0,0 +1,30 @@
+## Results
+
+### WenetSpeech char-based training results (Non-streaming and streaming) on zipformer model
```

Comment:
Could you update the title? I think it is about multi-dataset, not wenetspeech alone, right?

Also, it contains only non-streaming results and there are no streaming results yet.